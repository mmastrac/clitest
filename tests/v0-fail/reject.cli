#!/usr/bin/env clitest --v0

$ echo "hello"
reject {
    ! hello
}
*
