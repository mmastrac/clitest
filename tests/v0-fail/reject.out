Running <root>/reject.cli ...
printf "hello\n"
---
hello
---
ERROR: pattern reject at line <root>/reject.cli:5 does not match output line "<eof>"
matching Any(End)
  matching End
  -matching Literal(hello)
  -literal match: "hello" == "hello"
-matching Literal(hello)
-literal match: "hello" == "hello"

‚ùå FAIL

<root>/reject.cli FAILED
Error: pattern reject at line <root>/reject.cli:5 does not match output line "<eof>"

